00_Base/src/util/command.executor.ts

```diff
@@ -89,6 +91,7 @@ export class CommandExecutor {
     await this.responseUrlRepo.saveResponseUrl(
       correlationId,
       stopSession.response_url,
+      stopSession.session_id,
```

Comment:
Example of redundant transaction look up.

A few lines above this call we fetched transaction. But then in this saveRepsonseUrl method we seem to do another transaction look up.

Why not just extract the party id and country code from the transaction above and pass through into the method here?

Reply:
@langiole These feedback are very helpful. Thank you! I updated the code. Please let know if there is any further ideas.